packages/liveblocks-react/src/factory.tsx

```diff
@@ -175,19 +175,19 @@ export function createRoomContext<
       client.enter(roomId, {
         initialPresence: frozen.initialPresence,
         initialStorage: frozen.initialStorage,
-        unstable_batchedUpdates: frozen.unstable_batchedUpdates,
         shouldInitiallyConnect: frozen.shouldInitiallyConnect,
-      } as RoomInitializers<TPresence, TStorage>)
+        unstable_batchedUpdates: frozen.unstable_batchedUpdates,
```

Comment:
There was no need to "hide" this API, it's already a public API, and _needs_ to be public for all people on React 17 or lower. By making this public now, we no longer have to overwrite the TypeScript inference with a forced `as`-annotation. Had we done this before, this bug would have never existed.